app/screens/redeem-lnurl-withdrawal-screen/redeem-bitcoin-detail-screen.tsx

```diff
@@ -135,8 +135,8 @@ const RedeemBitcoinDetailScreen = ({
 }: StackScreenProps<RootStackParamList, "redeemBitcoinDetail">) => {
   const { callback, domain, defaultDescription, k1, minWithdrawable, maxWithdrawable } =
     route.params
-  const minWithdrawableSatoshis = minWithdrawable / 1_000
-  const maxWithdrawableSatoshis = maxWithdrawable / 1_000
+  const minWithdrawableSatoshis = minWithdrawable / 1000
```

Comment:
weird. this is valid `ts` afaik? probably a wrong setting somewhere in our tsconfig or somewhere else?

Reply:
Yeah I thought it was strange as well. It works locally.

Reply:
would be worth understand what settings are different but approving for now probably not worth the effort